website/content/en/docs/building-operators/golang/quickstart.md

```diff
@@ -1,500 +1,102 @@
 ---
-title: Golang Based Operator Quickstart
-linkTitle: Quickstart
-weight: 10
+title: Quickstart for Go-based Operators
+linkTitle: QuickStart
+weight: 20
+description: A simple set of instructions that demonstrates the basics of setting up and running a Go-based operator.
 ---
 
-**NOTE:** For the SDK versions prior to `v0.19.0` please consult the [legacy docs][legacy_quickstart_doc] for the [legacy CLI][legacy_CLI] and project.
-
-This guide walks through an example of building a simple memcached-operator using the operator-sdk CLI tool and controller-runtime library API.
+This guide walks through an example of building a simple memcached-operator powered by Go using tools and libraries provided by the Operator SDK
 
 ## Prerequisites
 
-- [go][go_tool] version v1.13+.
-- [docker][docker_tool] version 17.03+.
-- [kubectl][kubectl_tool] version v1.11.3+.
-- [kustomize][kustomize_tool] v3.1.0+
-- Access to a Kubernetes v1.11.3+ cluster.
+- [Install operator-sdk][operator_install] and its prequisites.
+- Access to a Kubernetes v1.11.3+ cluster (v1.16.0+ if using `apiextensions.k8s.io/v1` CRDs).
 
-## Create a new project
+## Quickstart Steps
 
-Use the CLI to create a new memcached-operator project:
+### Create a project
 
+Create and change into a directory for your project. Then call `operator-sdk init`
+with the Go plugin to initialize the [base project layout][project_layout]:
```

Comment:
Is there a link for this?

Reply:
Good point. I thought that we have that for all types. Also, should not the CI fail in this case?

Reply:
I think this syntax is interpretted verbatim when there isn't a link reference because there isn't an actual link to check.